Readability.js

```diff
@@ -197,20 +197,29 @@ Readability.prototype = {
       return pathBase + uri;
     }
 
-    function convertRelativeURIs(tagName, propName) {
-      var elems = articleContent.getElementsByTagName(tagName);
-      this._forEachNode(elems, function(elem) {
-        var relativeURI = elem.getAttribute(propName);
-        if (relativeURI != null)
-          elem.setAttribute(propName, toAbsoluteURI(relativeURI));
-      });
-    }
-
-     // Fix links.
-    convertRelativeURIs.call(this, "a", "href");
+    var links = articleContent.getElementsByTagName("a");
+    this._forEachNode(links, function(link) {
+      var href = link.getAttribute("href");
+      if (href) {
+        // Replace links with javascript: URIs with text content, since
+        // they won't work after scripts have been removed from the page.
+        if (href.indexOf("javascript:") === 0) {
+          var text = new Text();
+          text.textContent = link.textContent;
+          link.parentNode.replaceChild(text, link);
```

Comment:
This is expected. I tried to reduce this paragraph-creation logic (it's in the "experimental" block in Readability.js), but it caused a lot of test failures because things get weighted by how many paragraphs are in them, and obviously this is skewing the numbers. :-)